v2/googlecloud-to-googlecloud/src/test/java/com/google/cloud/teleport/v2/templates/bigtablechangestreamstogcs/Tools.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.templates.bigtablechangestreamstogcs;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.util.Base64;
+
+public class Tools {
+  private Tools() {
+    // don't create
+  }
+
+  public static String bbToString(ByteBuffer val) {
+    if (val == null) {
+      return null;
+    }
+    return new String(val.array(), Charset.defaultCharset());
```

Comment:
nit: `StandardCharsets.UTF_8.decode(val).toString()`

Reply:
Not sure if this suggested edit is any better, TBH